src/app/pages/contact/form.js

```diff
@@ -90,11 +91,15 @@ export default function ContactForm() {
         () => navigate('/confirmation/contact'),
         [navigate]
     );
+    const searchParams = new window.URLSearchParams(window.location.search);
+    const bodyParams = searchParams.getAll('body').join('\n');
 
     return (
         <SalesforceForm postTo={postTo} afterSubmit={afterSubmit}>
             <input type="hidden" name="external" value="1" />
             <input type="hidden" name="product" value={product} />
+            <input type="hidden" name="user_id" value={searchParams.get('user_id')} />
```

Comment:
can we populate this for osweb submissions as well? from `api/user`

Reply:
So have the form pull it if it's not supplied as a parameter?

Reply:
the cookie user is loaded already right? or no? i think we put the cookied user id in here if no user id is passed in the url

Reply:
I think user cookie is maintained in accounts, not directly visible to osweb.
osweb uses the accounts API to populate a context for user info. That's what I've got it using.

Reply:
i think you're understanding what i'm saying and i'm just saying it in a confusing way.

the cookied user is potentially different from the active user in assignable, because assignable uses a session managed by token passing based on the lti launch. so assignable will pass a user uuid to this form and it might be different from the "authenticated user" as far as osweb is concerned, and that is fine. we should use the passed id as a priority and fall back on the user in osweb's standard authentication as a fallback